os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -659,7 +659,7 @@ static int _tizenrt_create_task(struct task_struct *ptask, const char *name, u32
 		sparam.sched_priority = 206;
 	}
 	if (strcmp(name, "rtw_recv_tasklet") == 0) {
-		sparam.sched_priority = 205;
+		sparam.sched_priority = 206;
```

Comment:
Could you let me know why you set it over 200? Normal pri of apps is 100.

Reply:
@sunghan-chang  This priority setting has been made to achieve the 20Mbps throughput target for RX and TX. these priorities are for our wifi driver tasks.

Reply:
@prashantrar 
Other tasks may not work properly when Wi-Fi is running.
now, HPWORK priority is 201.

Reply:
Alright I have tested with the lower priority I.e. in the 100 range and performance is not affected. I will push it soon.

Reply:
I have fixed the priorities now